static/tools/mk/lint.mk

```diff
@@ -37,7 +34,7 @@ lint-unused:
 	@gometalinter $(LINT_EXCLUDE) --disable-all --enable=unused --deadline=45s  $(LINT_FOLDERS)
 lint-gocyclo:
 	@echo "lint-gocyclo"
-	@gometalinter $(LINT_EXCLUDE) --disable-all --enable=gocyclo --cyclo-over=20 --deadline=45s  $(LINT_FOLDERS)
+	@gometalinter $(LINT_EXCLUDE) --disable-all --enable=gocyclo --cyclo-over=16 --deadline=45s  $(LINT_FOLDERS)
```

Comment:
We should aim for 12 or even less. Our current top is 16 in one method, then it's 13 in one test file and 12 in a few.

Reply:
Sounds good.